sections/infrastructure.include

```diff
@@ -4431,7 +4431,7 @@
 
           <p class="note">This is a rather unusual low-level operation for which no primitives are defined by JavaScript.</p>
       4. Add <var>transferResult</var> as the last element of <var>outputTransferList</var>.
-  6. Return { \[[Clone]]: <var>clone</var>, \[[<var>transferList</var>]]:
```

Comment:
No, this is the [[Clone]] internal slot, which gets the value of the cloned content, it's not the [[Clone]] internal method. (Yes, I know there is obvious ambiguity here.)